crates/bevy_color/src/palettes/basic.rs

```diff
@@ -0,0 +1,38 @@
+//! Named colors from the CSS1 specification, also known as
+//! [basic colors](https://en.wikipedia.org/wiki/Web_colors#Basic_colors).
+//! This is the same set of colors as VGA.
```

Comment:
VGA should be spelled out, and a link would be nice.

Reply:
Added a link. I don't want to spell it out because the name "video graphics array" is actually confusing.